modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -3309,7 +3309,8 @@ public void setParameterBooleanFlagWithCodegenProperty(CodegenParameter paramete
             parameter.isPrimitiveType = true;
         } else if (Boolean.TRUE.equals(property.isFile)) {
             parameter.isFile = true;
-            parameter.isPrimitiveType = true;
+            // file is *not* a primitive type
```

Comment:
I am confused with the term `primitive`. (I did not notice `additional`.)

> An additional primitive data type "file" is used by the Parameter Object and the Response Object to set the parameter type or the response as being a file.

from http://swagger.io/specification/#data-types-12

To clarify, I want to know whether `additional primitive data type` is `isPrimitiveType` or not.